index.js

```diff
@@ -255,19 +256,21 @@ if (heroku.dockerBuildArgs) {
       }
     }
 
-    core.setOutput(
-      "status",
-      "Successfully deployed heroku app from branch " + heroku.branch
-    );
+    fs.appendFileSync(process.env.GITHUB_OUTPUT,
```

Comment:
I am pretty sure this is not solution.  Per the page I've listed below, the implementation of `setOutput` has been updated in actions/core version >= 1.10.0 

https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/#patching-your-actions-and-workflows

All that I think is needed to mitigate the issue to uprev the version of actions/core in package.json.